easybuild/easyconfigs/n/NWChem/NWChem-7.2.3-iofbf-2023a.eb

```diff
@@ -0,0 +1,51 @@
+name = 'NWChem'
+version = '7.2.3'
+
+homepage = 'https://nwchemgit.github.io/'
+description = """NWChem aims to provide its users with computational chemistry tools that are scalable both in
+ their ability to treat large scientific computational chemistry problems efficiently, and in their use of available
+ parallel computing resources from high-performance parallel supercomputers to conventional workstation clusters.
+ NWChem software can handle: biomolecules, nanostructures, and solid-state; from quantum to classical, and all
+ combinations; Gaussian basis functions or plane-waves; scaling from one to thousands of processors; properties
+ and relativity."""
+
+toolchain = {'name': 'iofbf', 'version': '2023a'}
+toolchainopts = {'i8': True}
+
+source_urls = ['https://github.com/nwchemgit/nwchem/archive/refs/tags/',
+               'https://www.chemie.uni-bonn.de/grimme/de/software/dft-d3/']
+sources = ['v%(version)s-release.tar.gz',
+           {'filename': 'dftd3.tgz',
+            'extract_cmd': 'cp -f %s %(builddir)s/nwchem-%(version)s-release/src/nwpw/nwpwlib/nwpwxc' }]
+patches = [
+    'NWChem-7.0.2_fix_gnumakefile.patch',
+    'NWChem-7.2.3-PR-1042.patch'
+]
+checksums = [
+    {'v7.2.3-release.tar.gz': 'fec76fbe650cdab8b00c8c1d7a5671554313e04a8e9e2fb300a7aad486910e6f'},
+    {'dftd3.tgz': 'd97cf9758f61aa81fd85425448fbf4a6e8ce07c12e9236739831a3af32880f59'},
+    {'NWChem-7.0.2_fix_gnumakefile.patch': '89c634a652d4c8c358f8388ac01ee441659e3c0256c39b6494e2885c91f9aca4'},
+    {'NWChem-7.2.3-PR-1042.patch': '5d69fd67976a5a3a6a0d14a444812605141d0c2bf65e4af97ace44f541b8bd9d'},
+]
+
+dependencies = [
+    ('GlobalArrays', '5.8.2'),
+    ('Python', '3.12.3'),
+]
+
+preconfigopts  = 'export EXTRA_LIBS=-lutil && '\
+                 'export BLASOPT=-lflexiblas && '\
+                 'export LAPACK_LIB=-lflexiblas && '\
+                 'export USE_SCALAPACK=n && '
```

Comment:
the BLASOPT/LAPACK_LIB/USE_SCALAPACK here is ignored, unfortunately.

You'll have to override the easyblock using:
```preconfigopts  = 'export EXTRA_LIBS=-lutil'
buildopts = 'BLASOPT=-lflexiblas64_intel LAPACK_LIB=-lflexiblas64_intel SCALAPACK=-lscalapack64 '
buildopts += 'EXTERNAL_GA_PATH=$EBROOTGLOBALARRAYS '
buildopts += 'COPTIMIZE="$CFLAGS" FOPTIMIZE="$FFLAGS" '
buildopts += 'USE_SCALAPACK_I8=y BLAS_SIZE=8 LAPACK_SIZE=8 SCALAPACK_SIZE=8 FC="$F77" #'
```
notice the `#` a the end, otherwise the easyblock stuff will come after, overriding our flexiblas64 etc. Alternatively I need to teach the framework about `libscalapack64.so`